apps/dashboard/src/model-assessment-vision/__mock_data__/visionModelAssessmentData.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  //IVisionExplanationDashboardData,
+  IDataset,
+  IModelExplanationData
+} from "@responsible-ai/core-ui";
+
+import { explanation } from "./explanation";
+
+export const visionModelAssessmentData: IDataset = {
+  categorical_features: [],
+  class_names: ["Water Tower", "not Water Tower"],
+  feature_names: [],
+  features: [],
+  predicted_y: [],
+  probability_y: [],
+  target_column: "",
+  task_type: "classification",
+  true_y: []
+};
+
+export const visionModelExplanationData: Omit<
+  IModelExplanationData,
+  "predictedY" | "probabilityY"
+> = {
+  precomputedExplanations: {
+    localFeatureImportance: {
+      featureNames: [explanation],
+      scores: []
+    },
+    textFeatureImportance: [
+      {
+        localExplanations: [
+          [1, 2, 3],
+          [1, 2, 3],
+          [1, 2, 3]
+        ],
+        text: [
+          "https://raw.githubusercontent.com/slundberg/shap/master/data/imagenet50/sim_n03029197_6381.jpg",
```

Comment:
you can just set this in a loop since it's the same thing repeated, instead of hardcoding it here